org-code-codebuilderapp/app/src/main/java/org/code/codebuilderapp/Destinations.java

```diff
@@ -0,0 +1,31 @@
+package org.code.codebuilderapp;
+
+public final class Destinations {
```

Comment:
(minor) Probably helpful to have a class comment here.  (We'll be able to generate documentation from the header comments and they also show up in IDEs.)

Reply:
✔️